dev/drivers/scripts/mesoscale/plots/jevs_mesoscale_headline_plots.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+#PBS -N jevs_mesoscale_headline_plots
+#PBS -j oe
+#PBS -S /bin/bash
+#PBS -q dev
+#PBS -A VERF-DEV
+#PBS -l walltime=10:00:00
+#PBS -l place=vscatter:exclhost,select=1:ncpus=128:ompthreads=1
+#PBS -l debug=true
+#PBS -V
+
+set -x
+export model=evs
+module reset
+export machine=WCOSS2
+
+# ECF Settings
+export RUN_ENVIR=nco
+export SENDECF=YES
+export SENDCOM=YES
+export KEEPDATA=YES
+export SENDDBN=YES
+export SENDDBN_NTC=
+export job=${PBS_JOBNAME:-jevs_mesoscale_headline_plots}
+export jobid=$job.${PBS_JOBID:-$$}
+export SITE=$(cat /etc/cluster_name)
+export USE_CFP=YES
+export nproc=128
+
+# General Verification Settings
+export NET="evs"
+export STEP="plots"
+export COMPONENT="mesoscale"
+export RUN="atmos"
+export VERIF_CASE="headline"
+export MODELNAME=${COMPONENT}
+
+# EVS Settings
+export HOMEevs="/lfs/h2/emc/vpppg/noscrub/$USER/temp/EVS"
+export HOMEevs=${HOMEevs:-${PACKAGEROOT}/evs.${evs_ver}}
+export config=$HOMEevs/parm/evs_config/mesoscale/config.evs.prod.${STEP}.${COMPONENT}.${RUN}.${VERIF_CASE}
+
+# Load Modules
+source $HOMEevs/versions/run.ver
+
+source /usr/share/lmod/lmod/init/sh
+module reset
+export HPC_OPT=/apps/ops/prod/libs
+export MET_bin_exec="bin"
```

Comment:
@MarcelCaron-NOAA  -- can you do this edit now in this PR? Otherwise, make sure you nab it in the follow on mesoscale plots PR.

Reply:
Sure!  Updated the whole thing to follow the other mesoscale plotting scripts in the parallel.